### DISCUSSION TITLE
Is Github Education available for Teacher's?

### DISCUSSION THREAD

**Discussion by markedtoolio** [community/community#143823] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to know whether teachers could register for github education?

**Comment by iamsaimehar** [comment]
Yes it is available for Teacher's, they will be requiring proper proof to register!

**Comment by M-tech-cmd** [comment]
Yes, teachers can register for GitHub Education! Here are the steps to apply:

1. Eligibility: You must be an educator, faculty member, or researcher with a verifiable school-issued email address.

2. GitHub Account: Ensure you have a personal GitHub account
 
3. Application: Go to the [GitHub Education page](https://education.github.com/) and click on “Get teacher benefits” under the “Individuals” section.

4.Verification: Provide proof of your academic status, such as a valid academic ID or other documentation that confirms your affiliation with an educational institution.

5.Submission: Complete the application by describing how you plan to use GitHub in your teaching.

[Once your application is reviewed and approved, you’ll gain access to various](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-teachers/apply-to-github-education-as-a-teacher)

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/143824#discussioncomment-11180513) for reference to why this discussion was closed and locked. Thanks!
